spec/acceptance/nodesets/default.yml

```diff
@@ -24,5 +31,8 @@ HOSTS:
         gpgkeys:
           - https://getfedora.org/static/352C64E5.txt
 CONFIG:
+<% if ENV['BEAKER_PUPPET_COLLECTION'] -%>
+  puppet_environment: <%= ENV['BEAKER_PUPPET_COLLECTION'] %>
```

Comment:
I don't see puppet_environment in any beaker-related gems.  Is this a typo?  Should it be puppet_collection?

Reply:
I have made the requested change to the nodesets to correct puppet_environment to puppet_collection.